cmd/edge-orchestration/main.go

```diff
@@ -69,8 +70,21 @@ const (
 	cipherKeyFilePath = edgeDir + "/user/orchestration_userID.txt"
 	deviceIDFilePath  = edgeDir + "/device/orchestration_deviceID.txt"
 	mnedcServerConfig = edgeDir + "/mnedc/client-config.yaml"
+
+	orchUserID = "Home Edge Network\n"
```

Comment:
Is it okay if we open this cipher key in public?

Reply:
Done!

Reply:
Done!